File: uri 1012.c
#include <stdio.h>
 
int main() {
 
    const double pi = 3.14159;
    float A,B,C,TRIANGULO,CIRCULO,TRAPEZIO,QUADRADO,RETANGULO;
    scanf("%f %f %f",&A,&B,&C);
    TRIANGULO = ((A*C)/2);
    CIRCULO = (pi*C*C);
    TRAPEZIO =(((A+B)/2)*C);
    QUADRADO = (B*B);
    RETANGULO = (A*B);
    printf("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f\n",TRIANGULO,CIRCULO,TRAPEZIO,QUADRADO,RETANGULO);
 
    return 0;
}

File: uri 1012.py
r = input().split(" ")
A, B, C = r
print("TRIANGULO: %0.3f" %(0.5 * float(A) * float(C)))
print("CIRCULO: %0.3f" %(3.14159 * float(C)**2))
print("TRAPEZIO: %0.3f" %(((float(A) + float(B)) / 2) * float(C)))
print("QUADRADO: %0.3f" %(float(B)**2))
print("RETANGULO: %0.3f" %(float(A) * float(B)))
